### PR TITLE
Add trying apps section

### PR DESCRIPTION
**Pull request by terencode**

### CODE REVIEW

File: download/index.html [hunk 1]
```diff
@@ -35,25 +36,25 @@
     </p>
     <pre class="language-markup">
       <code class="language-markup">
-[liri-unstable]
-SigLevel = Optional TrustAll
-Server = https://repo.liri.io/archlinux/unstable/$arch/
+        [liri-unstable]
```
Comment (plfiorini) [inline download/index.html]: don't indent `code` because this text is rendered verbatim

Comment (plfiorini) [inline download/index.html]: i can change it editing the commit

**Comment by plfiorini** [comment]
Merged

File: download/index.html (hunk 1 above)
Comment (terencode) [inline download/index.html]: Thanks.

Comment (plfiorini) [inline download/index.html]: you are welcome, thanks for the patch

**Pull request closed without merging**